build-version.sh

```diff
@@ -38,10 +38,25 @@ elif [ "$TARGET_ARCH" == "aarch64-unknown-linux-gnu" ]; then
     rustup target add "$TARGET_ARCH"
```

Comment:
This file has changed mode from 755 to 644. That's what's causing your CI failures. `chmod u+x build-version.sh` or something and commit the result?